<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oracle_locks_window</class>
 <widget class="QMainWindow" name="oracle_locks_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Oracle locks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icons/MGrep.ico</normaloff>:/icons/icons/MGrep.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <item row="4" column="2">
     <widget class="QComboBox" name="e_server_name"/>
    </item>
    <item row="15" column="5">
     <widget class="QPushButton" name="b_kill_session_lock">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Kill selected session</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/kill.gif</normaloff>:/icons/icons/kill.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="20" column="5">
     <widget class="QPushButton" name="b_kill_table_lock">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Kill selected session</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/kill.gif</normaloff>:/icons/icons/kill.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="l_server_name">
      <property name="text">
       <string>Oracle name server:</string>
      </property>
      <property name="buddy">
       <cstring>e_server_name</cstring>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="6">
     <widget class="QTableView" name="o_lst1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="6">
     <widget class="Line" name="line_1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLabel" name="l_table_name">
      <property name="text">
       <string>Table Name:</string>
      </property>
      <property name="buddy">
       <cstring>e_table_name</cstring>
      </property>
     </widget>
    </item>
    <item row="18" column="2">
     <widget class="QComboBox" name="e_table_name">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="6">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="6">
     <widget class="QTableView" name="o_lst2">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="18" column="3">
     <widget class="QPushButton" name="b_table_lock">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Search lock table</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/go.gif</normaloff>:/icons/icons/go.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="b_session_lock">
      <property name="text">
       <string>Check session lock</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/go.gif</normaloff>:/icons/icons/go.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QPushButton" name="b_load_list_table">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Load into combo box the list of tables</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/table.gif</normaloff>:/icons/icons/table.gif</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>e_server_name</tabstop>
  <tabstop>b_session_lock</tabstop>
  <tabstop>o_lst1</tabstop>
  <tabstop>b_kill_session_lock</tabstop>
  <tabstop>b_load_list_table</tabstop>
  <tabstop>e_table_name</tabstop>
  <tabstop>b_table_lock</tabstop>
  <tabstop>o_lst2</tabstop>
  <tabstop>b_kill_table_lock</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>b_session_lock</sender>
   <signal>clicked()</signal>
   <receiver>oracle_locks_window</receiver>
   <slot>slot_search_session_lock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_kill_session_lock</sender>
   <signal>clicked()</signal>
   <receiver>oracle_locks_window</receiver>
   <slot>slot_kill_session_lock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_table_lock</sender>
   <signal>clicked()</signal>
   <receiver>oracle_locks_window</receiver>
   <slot>slot_search_table_lock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>598</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_kill_table_lock</sender>
   <signal>clicked()</signal>
   <receiver>oracle_locks_window</receiver>
   <slot>slot_kill_table_lock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_load_list_table</sender>
   <signal>clicked()</signal>
   <receiver>oracle_locks_window</receiver>
   <slot>slot_load_table_list()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_search_session_lock()</slot>
  <slot>slot_kill_session_lock()</slot>
  <slot>slot_load_table_list()</slot>
  <slot>slot_search_table_lock()</slot>
  <slot>slot_kill_table_lock()</slot>
 </slots>
</ui>
